app/scenes/Settings/components/UserListItem.tsx

```diff
@@ -31,8 +31,8 @@ const UserListItem = ({ user, showMenu }: Props) => {
           ) : (
             t("Invited")
           )}
-          {user.isAdmin && <Badge primary={user.isAdmin}>t("Admin")</Badge>}
-          {user.isSuspended && <Badge>t("Suspended")</Badge>}
+          {user.isAdmin && <Badge primary={user.isAdmin}>{t("Admin")}</Badge>}
```

Comment:
@CuriousCorrelation Note I had to fix these lines

Reply:
Thank you, noted